src/core/util/log.cc

```diff
@@ -40,6 +40,8 @@ void gpr_unreachable_code(const char* reason, const char* file, int line) {
                    grpc_core::SourceLocation(file, line));
 }
 
+int absl_vlog2_enabled() { return ABSL_VLOG_IS_ON(2); }
```

Comment:
LGTM for this and the ruby changes. I'm just wondering if this should be named something like `gpr_absl_vlog2_enabled` since it's in the global scope.

Reply:
Addressing that in this PR - with other related functions.
https://github.com/grpc/grpc/pull/37431